src/main/resources/locale.properties

```diff
@@ -1,265 +1,265 @@
+command.roles.sub.managerRole.noRole=
+command.roles.sub.managerRole.set=
+command.roles.sub.remove=
+command.roles.sub.remove.notARepRole=
+command.roles.sub.remove.removed=
+command.roles.sub.removeDonor=
+command.roles.sub.removeDonor.remove=
+command.roles.sub.removeReceiver=
+command.roles.sub.removeReceiver.remove=
+command.scan.canceled=
+command.scan.canceling=
+command.scan.completed=
+command.scan.description=
+command.scan.error.history=
+command.scan.error.queueFull=
+command.scan.error.running=
+command.scan.progress=
+command.scan.result=
+command.scan.scheduling=
+command.scan.sub.cancel=
+command.scan.sub.cancel.noTask=
+command.setup.description=
+command.setup.dialog.channels=
+command.setup.dialog.channels.allChannel=
+command.setup.dialog.continueToProceed=
+command.setup.dialog.loadDefaults=
+command.setup.dialog.managerRole=
+command.setup.dialog.roles=
+command.setup.dialog.rolesFormat=
+command.setup.dialog.selectLanguage=
+command.setup.dialog.setupComplete=
+command.setup.dialog.starting=
+command.setup.dialog.welcome=
+command.thankwords.description=
+command.thankwords.error.patternNotFound=
+command.thankwords.sub.add=
+command.thankwords.sub.add.added=
+command.thankwords.sub.check=
+command.thankwords.sub.check.confidence=
+command.thankwords.sub.check.match.answer=
+command.thankwords.sub.check.match.fuzzy=
+command.thankwords.sub.check.match.mention=
+command.thankwords.sub.check.match.noMatch=
+command.thankwords.sub.check.notConfident=
+command.thankwords.sub.check.reference=
+command.thankwords.sub.check.result=
+command.thankwords.sub.list=
+command.thankwords.sub.list.list=
+command.thankwords.sub.loadDefault=
+command.thankwords.sub.loadDefault.added=
+command.thankwords.sub.loadDefault.available=
+command.thankwords.sub.remove=
+command.thankwords.sub.remove.removed=
+command.top.title=
+conversation.cancel=
+conversation.canceled=
+conversation.exit=
+conversation.finished=
+conversation.inProgress=
+conversation.start=
+error.commandNotFound=
+error.invalidArguments=
+error.invalidChannel=
+error.invalidMessage=
+error.invalidNumber=
+error.invalidRegex=
+error.invalidRole=
+error.missingPermission=
+error.missingPermissionChannel=
+error.missingPermissionGuild=
+error.noValidChannel=
+error.notABoolean=
+error.notANumber=
+error.notYourEmbed=
+error.prefixTooLong=
+error.roleAccess=
+error.userNotFound=
+listener.messages.request.descr=
+listener.messages.request.descrThank=
+listener.messages.request.title=
+listener.reaction.confirmation=
+listener.reaction.removal=
+message.welcome=
+messages.destruction=
+selfCleanup.leave=
+selfCleanup.leave.title=
+selfCleanup.prompt=
+selfCleanup.prompt.title=
+thankType.answer=
+thankType.fuzzy=
+thankType.mention=
+thankType.reaction=
+word.continue=
+word.done=
+word.skip=
+words.code=
+words.language=
+words.level=
+words.link=
+words.message=
+words.name=
+words.refMessage=
+words.reputation=
+words.role=
+words.unknown=
+thankType.embed=
```

Comment:
Why are all empty?

Reply:
because the blank file is only used as key reference.
The fallback file is the english locale. The values of this file are never actually read.